sandbox/cgroup/inotify/LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2009 The Go Authors. All rights reserved.
```

Comment:
It's possible we'll want to use inotify outside of the cgroup code. What do you think about putting this code at `osutil/inotify` instead?

Reply:
Good point... Done.